app/Models/V2/Projects/Project.php

```diff
@@ -124,6 +126,16 @@ class Project extends Model implements MediaModel, AuditableContract, EntityMode
         'long',
         'answers',
         'ppc_external_id',
+        'detailed_intervention_types',
+        'proj_impact_foodsec',
+        'pct_employees_marginalised',
+        'pct_beneficiaries_marginalised',
+        'pct_beneficiaries_men',
+        'proposed_gov_partners',
+        'proposed_num_nurseries',
+        'proj_boundary',
+        'states',
+        'proj_impact_biodiv',
```

Comment:
Not a huge deal because it'll even out in the end, but if the branch you're merging this into also got updated from staging, the diffs in this file should go away.

Reply:
removed